src/providers/go-to-definition.ts

```diff
@@ -0,0 +1,23 @@
+import * as vscode from "vscode";
+
+export class MF6DefinitionProvider implements vscode.DefinitionProvider {
+  public async provideDefinition(
+    document: vscode.TextDocument,
+    position: vscode.Position,
+  ): Promise<vscode.Location | vscode.Location[]> {
+    const wordRange = document.getWordRangeAtPosition(position);
+    if (!wordRange) {
+      return [];
```

Comment:
```suggestion
      return null;
```

I know that `[]` is compliant with the required return type but it doesn't make sense. It's make more sense to convey that you are returning nothing. But this is just a recommendation.

Reply:
Adapted the changes as suggested